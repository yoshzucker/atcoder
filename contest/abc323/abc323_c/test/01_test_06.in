97 92
2300 1100 1800 800 1000 1500 2300 2200 1200 2200 1300 700 1600 900 500 1400 1400 2000 1400 600 1800 800 1400 1400 2300 2100 1500 2500 2000 1000 1900 1200 2200 1800 1900 2300 2100 2500 1300 500 2000 1900 1000 1500 700 2400 1900 1700 2200 1000 1900 700 800 1400 2400 1900 700 1900 600 900 600 500 1100 2000 600 2000 2300 2500 2100 1800 1000 2500 700 1700 1600 2200 900 1500 1000 2000 1500 2500 2500 2000 2400 1300 1200 2100 700 800 900 1000
oooxxoooooxoxxooxoxooxxxoxxooxoooxxooxoooxxooxxxoooxxooxxoooxxoxooxxoooxooxxxxoooxxxoooxxoox
xoxxxxxooooxxoxoxoooooxxooxxxooxxooxxoxooxoxxxxoxxxxooxxxxoooooooooooxxxoxxxxxxxoxxooxxooxoo
oooxxoxoooooxxxxooxxxxxoxxxoxoxooxxxxoxxooxxooxooxxxoxoxoxoxxoxxooooxoooxxxxxooxxxoxoooxxooo
xooxoxxxxoxoxxoxoxoxoooxxooooooooxxoxoooxoxxxxooxoxxoxoxoxoxoxxxxxxxoooxoxxxooxxooooxxoxxoox
ooxxxxoooooxxxxxoooxxxoxxooxooxooxoxoxxxxooxoxxoxoooxxxooooooxoxooxoxxooxoooxoooxoxxoxxoxxox
oxxxxxxoxxxxoxxoxooxxooxoxxxxoooxxoooooxoxooxooxoxoooxxooooxxoxxxoxoxxoooxxoxxxxxoxoooooxxox
xxxxoooxoxoooooxoooxxxxxoooxxoxoxoxooxxooxoxoooooooooooxxooooxoooooxooxooxxxooxooooooxoxxxxo
xooxoxoxxooxoxoxoooxxoxoooooooooxxxooxxoxxooooooooxxxoooxxxooxooooxoxoxoxooxxooooxoxxxooxxox
oxoooxxxxxxoxxxoooxxxoxoxooxooxoxooxoxoooxoxxoxxxxooxoooxxooxxoxxoxxxxoxxooxxooooxxoxxxxoooo
xxoxxoxxxxxooxooxoxxoooxoxxxxoxxooxxoxoxoxoooxxxxoxxoxoooooxxooxoooxoxxxxoooooxxoooooxxxxxoo
xoxooxoxxxxxxxxxxooooxoxooooxxoxoooxooxoxxoxooxoooooxoooxoooxoxoxxoxxoxxxxxxxxxoooooooxxxxxo
oxxxoooooooooooooxxoooxoooxoxoxoooxxoxoxooxxoxxoxoooooxoxxxxooxooxooxoxoooxxoxooxxxxooxoooxo
oooxxooxxoxxxxxxxoxxxxoxxoooxxxoxoooxxxxxooxoxxxxooxxoxoxxoooxoxooxxooxooxxxxxxxoxooxooxxxxx
xoxooxooxoxoooxooooxooxoooxxoxooooxxxxxooxoooooooooxooxoxxoooxxxxoxoxxxoxxoooooxoxoxxxxxoxxo
xxooxxooxoxooxoooooxoxxxxxoxxoxooxxooxxoooooxoooxoxxoxxoxxoxxxxxxxoxoooxoxxxoooooxxoxooxoooo
xxxxoxxooxoxxoxoooooxoooxxxoooooooxxxoxoxxxoooxxxxoxoxxoxooxxooxxooxoooxxoxooxxooxoxxoxooxxx
xoxoxoxxxoxxxoxoxxxxooxxoooooxoxxooxxxooxooxoooxxxooxoxoxooxxoxxooooxxxxxoxoxxoxxxoxxoxoxooo
oxoxooxxooxxxoooxxxxoxoooxxxooxoooooooxooxxoxoxxxxoooxxoxxxooxxxxoxxxxoxxxxoooooxxooxxxoxxox
oooxxoooxxxxooxxoxoooxxxoxooxxxxxxxoxxoxooxooooooxxxoooxooxxxxxxxoxxooxxoxoxxoxoxxoxoxxxxxxx
xxooxooxooxxoxxoxooxxxxxooooxooooxoxxoooxxxxxooxxxxxoxoxoxooxxoxoxxxxooxooxxxxxoooxxxxxxxxoo
oxoxoooxooxoooxoxoxxoooooxoxoxoxoooooooxxxxxxxxxoxxoxoxoxxxxoooxxxxooxxxoooxxxxooxoooxxoooxx
xxxooxooooooxxxxxxxxxxxxooooxoooxxoooooxooooooxoxxoxxxxoxxooooxxoxooxooxoxxoxoooxxoxxxxoxoox
ooxoxxoxxoxoxoxxoooxxxoxoxxxxoxooooxxoxxxooxxooxoooxxxxxoxoxooxxoooxoooxooxxxxxoxxxxxxxooxoo
ooooooxoxxxxoooooxxxxxxooooxxoxxooxxxooxooxxxoooooooxoooxoxoxooxoxxoxxoxxooxooxoxxxxoxxoooox
oxoxoooxxxxooxoxoxoxoxoooxxoxxxxooxxxxoxoxooxxoxxooxxoxxxoooxxoooooxoooooxoooxxoxxxooxooooxx
xoxxxxxoxoxoxxxxooxoxoxxxoxoooooxxoxxooooxxooxxxxoooooxxoxoxxxxoxoooxxxooooxxxooxxxooxooxooo
oxxxoxxoooooooxxoxxxxoxoxoooxxxoxoxxooooooxooxxoxoxxooooooxooooxoxxooxoxxoxooxoxoooooxxooxox
xoxxxxxxxoxxxxooxxoxxoxoooooxxxxxxoxoooxxoxoxoooooooxoxooxxxxxxxoxoxxoxooxxxoooxooxxooooooxx
xoxxoxoxxooxxoxxooxxoxxoxoooooxxoooxxoxxoxxoxoxoooxxxooxxoooooxxoxxoooxxoxxxoxxoxxxoooxxoooo
ooxxxxxxoooxooxxoxxooxxoxxoxoxooxoxxoxxoxxooxxxxooxxooxxxxoxoxoxxxxxooxxoxoxxxxoxxoxoxoxoxox
xxooooxxxxxxooxoooooxxoooxxxoooxxooxxoooxooxxoxxxoxxoxoxxoxooooxooxoxoooxxoxxoxxxooxxxxxxxoo
xxxxoxxooxxxoxoxxxoooxxxoxxoxoooxooooxxxxoooooxoxxoxxxxoooooxxxxoxooxxxoxxooxxxxxxxxooxoooxo
xxoxxxoxooxoxxxxoxooxxxoxxxoxooxxxooxxxoxoxxooxoooxxxxxoxxxooxxxxoxxoxooooxxxoxoxoxoxxoxxooo
oxooxooooxxxoxxxoooxxooxoxooooxxooxxoxxxoxoxxxoooxoxxxoooxoooooooooxoxooxxoxooooxxxooxoxoooo
xxxooooooxoxxxoooooxoooxoooxxoooxxxxxxxooooooxooxoxoxxoxxxxxxxooxoxxooooooxoxoooxxxooxxoooox
oooxoxooxxoxoxoxxxxoxxoxoooxoxoooooxoxxxooxxoooooooooxoxooxxxooooxxxxoxxooxooooxxxxxoooxxooo
xxxxxoxoooxoxxxxxxoooxooxoxxxxooxoooooxoooxxooxxxoxxxxoxxxxxoxoxooxoxoxxxxxooxxoxxxoxxooxxxx
oxoooooxoxxooxoxxxxoxooxxxoxoxoxoxooxooxoooxxoxxxxxoxxxxxoooxooxxxoxooxxxxxooxoxxoxxxoxoxoxx
ooxoxooooxoooxxxoxoooxoxxxoxoooxxoxxxxoxxxooxoooooxooxooxxxxxxxoxxxoxoxoxxxxoxoxxoxxxooxoooo
oxoooxoxxxxxxoxoxoxxooxxoxxxooxoxoooxooxxxooooxooxoooxoxoxxooooxxxxxxoooxxxxoxoxxooxoxooxooo
oxxoxxoxooxxxoxxxooxxxxxooxoxoxooxoooooooxxxoxoooxxxooooxxooxooxoxoxxoxxoxoooxoxooooxoooxxoo
xooxoxxoxoxxoxxoxxooxxxoxxxoooxooooooxxoxxxoxooooxxxxoxxoxxxxoxxxoxxxoxxxxoxooxxooxoxxxooxox
xoxoxoooxxoxoooooxoxxoxxoxooxoxooxooxoooxxoxoxxxoxxooxxooxxxooxoxxoooxxoxxoxxxoxxxoxxoxoxxxx
ooooooxxxxooxooxoooxoooxoooxoxxxooxxxxoooxooxxoxooxoxoxoxxxoooxoooooxxoxxoxxxooxxoxxoooooxox
xoxoooxxxoxxxoxoxooxxxxxoxxoxxxoxoooxoooooxoxxxxoxooxxoxxxooxoxxxxooxoxoxoxoxooooxxooooxooxo
ooxooxooxxoxxxooxxoxoxxooxxooooooxoxooxxxxooxxoxooxooooxxoooxoxoooxxxxxooxxoxoxxoxxxooxxooox
xxxoxoxooxoxxxxxoxoxooxoooxoooxoxooxxxxoooxoxxxxoxooxxooooooxxxoxxxoxoxoxoxoxooxxoxxxxxoooxo
xoxxxooxoxoxxxoooxoxooxxooooxxoxxooxxoxoxooxxooooxoxooxxxxxoxoxxoooxxxooxoxxooooooxxxxoxxxxx
xoxxooxoxoxxxooooooxxoxxoxooooooxxxooooooxooxoooxoxoxxoxxoxoooxxxxoooxxoxoooooxxoooxoooooxox
ooxoxooxooxooxxxoxxxoxooxooooxooxoooxxxoxxxoxooxooxxxxoooxxoooxoxoooxxxxooxxxxoxooooxxxoxoxx
ooxxooxoooxoxoxooxxoooooxooxoxxxooxoxooooooxooxoxoxoxxooooxoxooooxoooooxooooooxoxxoxoooxoxoo
oxxxoxoooxoooxxxooxxooxooxoxxoooooxoooxxoxoxoxoxoxxooxxoxxxoooxoxxxoxxoooxoooxoooxoxoooxxooo
oxoxoxxoxxoxooxooxxxoxxxoxxxooxoxxoxoooxooxxooxooxooooooooxxxooxxxxoxxooxooooxxoxxxxooxoooox
xxxxxxxxxoooxoxooxooxxoooxxooxoxoxoxoxxxoxoooxoooxoxxxxxxoxxooxxxoxooxooxxoooxoxxoxoxoooooxx
xxoxoxxxxxooxxoooooxoxxxxoxoxxxoxooxxxoxoxxooooxxxoxxoxoxxxxxxoxxxoooooxoooxxooxxxoooxooxooo
oxxxoooxxooxxxxxxxxxxoxxoxxoxooooooooxooooxoxoxooxxoxooxoooooxoxooxxxxxxoxxoxxxxoxooxoxxoxxo
ooxxxxxoooxxxxoooxoxooxxxxoxoxxxoxxoxoxxoxxoxxxoooxoooxxxxooxoooooxxooxxxoxxxxxoxxxooooxxooo
xoxxooxxxxxxxxxxoxoooxxoooxooxoxxxxoooxoxxooxoxoooooooxxooxxoxooooooooxxoxooxxxooxooxoxxoooo
xoxooooxooxxoxoxxoxoooxxxoooxooxoxoxxxxoxoxoooxxxooxoxxxxooxxoxxooxoooooxoooxxxxxxoxooxxoxox
xxooxxoooooxooooxoooxooooxxxooxooxoxxxooxxoxxxxooxxooxoxooxxxoxoooxxxxoxxxoxxxooxoxxoooxxxox
ooooxxoooxooxxooooooxxoooxooxxxxoxxxxxxooxoxoxxoxxxoooooxooooxoxxoxxxoooooxxoxxxxxooxxoxxxoo
xxxoxxoxoxxooxooxxxoooxxoxxooxxoxoooxooxxxooooxooxoooxoxxxxxxoxoooxooooxooxoxxooxxoxxooxxxoo
xxxoxxoooxooxoxooxxxooxxoxooxxxoooooxoooxoxxxxoooxoxxoxxoxxoxoxooxooxoxxooooooxxxxxxooxoxxxx
xxooxoxxoxxooxoxxooxxxooxxoooooxoooooxxoxxxoxxoxoxxxoooxxxxxoooxxxooooxxoxoxxxxxxoooxxoxxooo
xxoooxxooooooooxxooxoxxoxooxxxxoxxxoxoxooxooxoxoooxxxooxoxoxxoxxoxxooxoxoooxoxoxoxoooxoooxxo
oxoxxooxxxxxxoxoxooxxxoxooxooxooxoxxxoxxooxoxxxoooxooxxoxxxxxxxxoxxooxxoxooxxxoxoxoxoooxoxxx
xxxoxoooxoooxooxoooxooxxxxoxooooxoooxooxxoxoxxoooxoxxxooxoxxxooxxxoxoxoooxoxxxooxxxoxoooxxxo
xooxxoooxoooooxoooxoooxooooxoxxxxxxxoxxoxooxxxoxoxoxxxxxoooooooxxxxxxxxooxoxooxoxxxxxxoooooo
xxoxxxoxxxoxoxoxoxoooooxxoooxoxxoxxxxoooxxxooxooxooxoooxooxxxooooooxxooooxxxxxxooxxoxooxxxox
xoooxoxoxxxoxooxxxxxoxxxxxxoxoxxxooxxoxxoxxxoxoooxxoxoxoxxoxxooxxoxxxxxooxxxooooxxooxooxoooo
xxxxxxooxxooooxoxxxoxxoxoooooxoxxooxxoxxooxxxoxxxxxxooxxxxoxxxooooxxxxxxxxxxxoxoxxoxoooxxxox
oooxoxxxoxooooxxooxxoxxxooxoooooooxooxoooooxxxxoooxooxxooxooxxoxxoooxoxxxoooxoxooooxoooxxxxx
xxoxooxxxxooooxxxxoxxooooxoxooooooooxxoxoxxxoxxoxoooxooooooxooxooxxooooxxooxxxoxxoooxxoxxxxx
ooxxxoxxoooooxxoooxoooxxoxoooxxoxxoxoxoooooooxoxxxxoxoxxoooooxooooxxooxooxoxoooxoxoxoooxxxox
oxxxxxxxxxxxooxoxoxoxoxoxxxooxoxxxxoooooooooxxoooxoooxoxxxxoxoxxoooooooxooxoooxxooooxoxoxxox
oxxxxxoxoxooxxoxxoooooxxxxxxxoooxxooxooooooxxxxxoooxoxxxoxxoxoooooxxoxxooooxoxxxoooooooooxxx
oxoxxoxxoxxoxxoxxoxxooooxoxxoxxxxoxxxxoxoxoxxxoxxooxoxooxxxoxxoooxoxooxoxxxoxooxoooxooxoxxoo
xoxxxxooxoxxxxxxoxxoxoxxoxxoooxoxxxoxxooxxooxooxoxxxxoxxooxooxxxoooxxoxooooxxxooxxxxoxxooooo
xxooxxooxxoooxxxxooooxoxooooxxooooooxoxxxxxoxooooxxooxooxoooxoxxxxxoxxoooxoxoxoxxxxxxoxxooxx
oooxoxooxoooxxooooxxoxxxxooooxoxxxooxoooooxxoxxxoxooxxxxxoxxxoooxxxxooxxoxxxxooooxxooxxoooox
oxxooxxxxxoooxxxxoooooooxooxoxxxoxooooxoxooxoxooxoxoxoxooxxxxooxxoxxooooxoxoooooxxxoxoooooox
ooxxxooxxxooxxxoxoxoxxxoxxxoxxxxxoxoooxxxoxxxooxoxoxoooxxxxoxxxoxxoxxxoooooxooxooooxxxooxxoo
xxxoxxxoxxxoxxoxoxxoxoxooxxxxoxxoxxxxxxxoxxxxxxxoxxxxxooxoooxooxoxxoxooxxxoxoxxxxxxxoxxoxoxo
xxxxxoooxooxxxoxooxoxxxxxxoxxxxooxoooxxxxxoxoooooooxoxooxooxxoxooxxoooooxooxoxxxxxoxxxooooox
xooooooxxxxoxxooxxxoxoxxoxoxxoooooxoxoooooxooooxooxxxxooxxooooooxoxoxoxxxxxxxxooooxxxxoxooox
xooxxoxoooxoooxooooxxxxoxxxoxooxoxoxxxxoxoxxxxxxxxxxooxxoxxooooxxooxoxxxoooooxoxxxxxooxoxxxx
oooooxxoxxooxooxxxxoxoxxxxxxoxxxxoxxooxooooxoooxxxooxxoxxxooxxoxooxxxoooxooxxxxooxxoxxoooxox
xxxoooxxxoxoxxxooxxxooxxooxxxoxoxxooxoxxxoxoxoxooooxoxxxooooxoooxoxxxxxxooooxoooxxooxoxxooxx
oooxxxooxxxoooooxxxxoxooxxxxxoxxxoxoooxooxxxoooooxxxooxooxoxxoxxooooxxoxoxxoooxxxxooxxxxxxxx
oooxxoxxxxxoxxoxoxxxoxoxxxxxxxoxxoxxoooxxxxxxxoxoxoxxxoxoxxooxxxxoxoxooooxxoxxooooxxxxooxoxo
xooooxoooxoxxxxooxxxoxxxxxxoxxxooxxooxxooxxxooxoxoxxxxoxxxooooxxxooxoxxxxoxooooxooooxoxxxoox
xoxoxxxoxooxoxoxxoxoxooooooxoxoxxxoxxxxoxoooxxoxxxooxoooxoxoxoxooooooxxxoxoxooooxxxoooxxxxxo
oxoxoxoooxxoxxoxxxxoooxoxoxxoxxoxxxxoooxxxxxoxxoooxooxxxxoxoxoooxoooooooxoooxoooxxooxxxxooox
xxxoxxooooxoxxxxoxooxxxoooxooxooxxxoxxxooxoxooooxooxoxxooooxoxxoooxoxooxoxoxoooxoxxxxxooxoxx
xxxoxxxooooxooxxxxxxoxxooxxoxxooxxxxoooxxoxxxoxxoxxoooxxxxoxxxxxoxoooxoxooxxxooxoxxooxoxoxxx
ooxooxooxxxoxxxxxxxxxooooxooxxoxoxoxoxoooxoxxooooxxxoooxxoxoooxxxxxxxxxoxxoxxooxoxoxxxxxoooo
ooooooxxooxooxxoxoxoxxxooooxxxoxxxoxoooxxooxooxxxxoxxxxooooxooxxooxxxooxooxooxxxooxxxxoooxxx
