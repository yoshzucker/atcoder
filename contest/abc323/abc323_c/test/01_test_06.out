6
12
11
12
7
9
2
3
13
11
7
4
15
6
9
10
12
11
11
14
7
8
13
7
7
11
5
11
9
17
11
13
15
4
7
6
14
13
11
10
6
13
12
7
9
9
10
10
2
7
0
5
7
10
11
10
13
6
10
11
10
9
9
11
8
12
7
11
11
16
16
3
8
6
7
9
11
14
9
7
7
11
23
9
9
14
13
11
12
16
12
9
9
10
15
13
10
